### PR TITLE
 promoting runtime testcases to conformance suite

### PR DESCRIPTION
**Pull request by mkurylec**
a) ported  eligible conformance testcases into from e2e_node into e2e/common.
b) added them to the conformance golden list,
c) left  standard tests in node.

[reference issue](https://github.com/kubernetes/kubernetes/issues/67103) for visibility

### CODE REVIEW

**Comment by spiffxp** [comment]
/uncc @bgrant0607 
/cc @wangzhen127 
For the /lgtm (LGTM but I'd like to have you confirm the split looks good)

**Comment by spiffxp** [comment]
Hm, in that case
/lgtm
/hold
I'll remove the /hold when I get feedback from @wangzhen127 here

**Comment by spiffxp** [comment]
/release-note-none
/lgtm cancel
all valid points, thanks for the review @wangzhen127

**Comment by mkurylec** [comment]
To run tests under  k8s folder
**e2e (common)**
- `go run hack/e2e.go -- --build`
- configure a cluster accordingly (in this case was local)
- `go run hack/e2e.go --  --test --provider=local --test_args="--ginkgo.focus=blackbox test"`

**e2e_node**
- `make`
- `output/bin/e2e_node.test --ginkgo.focus "blackbox test"`

**Review by wangzhen127 — COMMENTED** [review]
I saw there were a few updates. So I took another pass.

Do you plan to remove test/e2e_node/container.go in this PR or in a follow-up PR?

**Comment by mkurylec** [comment]
@wangzhen127 performed all changes, let me know if itś ok, thanks! :)

File: test/e2e/common/runtime.go [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	gomegatypes "github.com/onsi/gomega/types"
+)
+
+const (
+	ContainerStatusRetryTimeout = time.Minute * 5
+	ContainerStatusPollInterval = time.Second * 1
```
Comment (wangzhen127) [inline test/e2e/common/runtime.go]: Nit: Move those 2 to test/e2e/common/container.go.

**Review by wangzhen127 — COMMENTED** [review]
/lgtm with one nit.
Feel free to squash the commits.

File: test/e2e/common/runtime.go (hunk 1 above)
Comment (mkurylec) [reply test/e2e/common/runtime.go]: sure, will do right away, thanks

**Comment by spiffxp** [comment]
/assign @mgdevstack 
This will need to be split up into two PR's, one to introduce the tests as e2e, and another to promote to conformance.  I suggest opening up new PR's and linking to this.

**Comment by mgdevstack** [comment]
@wangzhen127, I have opened new PR #67571 to port these container-runtime e2e_node test into e2e/common pertaining all your reviewed/requested changes along with new minor updates mentioned in new PR. Once #67571 gets lgtm/approval, I can open up new PR to promote them for Conformance and we can close this PR.

**Comment by mgdevstack** [comment]
https://github.com/kubernetes/kubernetes/pull/67571#issuecomment-414406310 needs to be addressed while we are porting from e2e_node to e2e. This would help us to promote them for conformance in #67612 .

**Pull request closed without merging**